src/pages/applicants/index.tsx

```diff
@@ -24,8 +24,11 @@ const Applicants: NextPage = () => {
   return (
     <>
       <Head>
-        <title>Ethereum Ecosystem Program | Applicants Overview</title>
-        <meta name='description' content='Applicants Overview' />
+        <title>Applicant Overview | Ethereum Foundation Ecosystem Support Program</title>
```

Comment:
Hmmm not sure if this will work. Title tags [should be 50-60 characters max](https://moz.com/learn/seo/title-tag), this one is 66 characters.

Reply:
@pettinarip @samajammin maybe we can use just `Ethereum Foundation ESP` instead of the full name?

Reply:
I'd suggest `Ethereum Ecosystem Support Program` for now but I'll ping the ESP team on this.

Reply:
LMK what you think @nhsz @pettinarip - I suggest we make those updates before merging.

Reply:
Perfect! will use `Ethereum Foundation ESP` as discussed in the TG chat.

Reply:
We still have the thank you pages that are exceeding the max title length. E.g. `Thank you for applying to Office Hours | Ethereum Foundation ESP`

One possible solution is to just print `Thank you | Ethereum Foundation ESP`, what do you think? these thankyou pages are not indexable so...I think it wouldn't matter too much.

Reply:
Implementing the shorter version: `Thank you | Ethereum Foundation ESP`